src/commands/rage.ts

```diff
@@ -0,0 +1,63 @@
+import supportsColor from 'supports-color';
+import { getBorderCharacters, table } from 'table';
+
+import BaseCommand from '../lib/baseCommand.js';
+import configstore from '../lib/configstore.js';
+import getCurrentConfig from '../lib/getCurrentConfig.js';
+import isCI, { ciName, isGHA } from '../lib/isCI.js';
+
+export default class RageCommand extends BaseCommand<typeof RageCommand> {
+  static description = 'Prints information for debugging.';
+
+  static hidden = true;
+
+  async run() {
+    const { apiKey, email, project } = getCurrentConfig.call(this);
+
+    const rage = {
+      CLI: {
+        Version: this.config.version,
+        // This `supports-color` logic is the same that `@oclif/core/ux` uses for `colorize` they
+        // just don't expose their internal `supportsColor` library.
+        'Color support': Boolean(supportsColor.stdout) && Boolean(supportsColor.stderr),
+      },
+      Platform: {
+        'CPU Architecture': process.arch,
+        OS: process.platform,
```

Comment:
this _might_ make it easier for us to write tests for, but not sure

```suggestion
        'CPU Architecture': this.config.arch,
        OS: this.config.platform,
```

Reply:
honestly i wonder if the entire `this.config.versionDetails` object might be worth dumping here